tests/conftest.py

```diff
@@ -7,14 +7,18 @@
 
 CCDS_ROOT = Path(__file__).parents[1].resolve()
 
-args = {
+base_args = {
     "project_name": "NestaTestCookie".lower(),
     "author_name": "Nesta",
     "repo_name": "nestatestcookie",
+    "openness": "public",
 }
+args_list = [
+    {**base_args, "venv_type": venv_type} for venv_type in ["uv", "venv", "conda"]
+]
 
 
-@pytest.fixture(scope="class", params=[args])
+@pytest.fixture(scope="class", params=args_list)
```

Comment:
Currently, test parameter name/id don't show in pytest runs. We could use venv_type for the [parameter name/id](https://docs.pytest.org/en/stable/example/parametrize.html#different-options-for-test-ids) in the test. Helpful for troubleshooting which parameter went wrong.

```suggestion
def get_test_id(test_param):
    assert isinstance(test_param, dict)
    assert "venv_type" in test_param

    return test_param["venv_type"]
    
@pytest.fixture(scope="class", params=args_list, ids=get_test_id)
```